Apoorv38.md

```diff
@@ -0,0 +1,52 @@
+ ---- 
+ name: "@Apoorv38"
+ project: "Mech-pupp"
+ ----
+
+# Mech-pupp
+
+## Summary
+
+I'm going to build a quadruped puppy using servo motors attached to 3D printed legs. This puppy will be controlled using arduino boards. It always   follows its owner without running into stuff.
+I am excited to build this one. Never made something like this.
+I think that everyone would like to have a freindly companion who would accompany you where ever you go and doesn't require much maintanence.
+I'm not sure if it would be as good as I think but it's worth trying.
+
+## Plan
+
+In each leg I will use 3 servo-motors one for controlling the direction of   the pupp, the other two are for forward motion.
+I will use IR proximity sensor and Ultrasonic distance finder to avoid   obstalcesfrom a safe distance.
+I will use esp 32 cam for face recognition of the owner.
+I'll use OLED screen for its face.
+It would be powered using a Lithium Polymer battery(800 mAh).
+I am using two Arduino boards because I think 1 wouldn't be enough for the pupp.
```

Comment:
How are the 2 Arduino boards going to communicate with each other? Are they networked?

Reply:
Using software serial.